addons/l10n_vn/i18n_extra/vi_VN.po

```diff
@@ -25,72 +91,110 @@ msgstr "HHDV Mua vào"
 msgid "Sales of Goods and Services"
 msgstr "HHDV Bán ra"
 
+#. module: l10n_vn
+#: model:account.report,name:l10n_vn.tax_report
+msgid "Tax Report"
+msgstr "Báo cáo Thuế"
+
+#. module: l10n_vn
+#. odoo-python
+#: code:addons/l10n_vn/models/res_bank.py:0
+#, python-format
+msgid ""
+"The QR Code Type must be either Merchant ID, ATM Card Number or Bank Account"
+" to generate a Vietnam Bank QR code for account number %s."
+msgstr ""
+"Loại mã QR phải là ID người bán, số thẻ ATM hoặc tài khoản ngân hàng"
+" để tạo mã QR Ngân hàng Việt Nam cho số tài khoản %s."
+
+#. module: l10n_vn
+#. odoo-python
+#: code:addons/l10n_vn/models/res_bank.py:0
+#, python-format
+msgid ""
+"The proxy type %s is not supported for Vietnamese partners. It must be "
+"either Merchant ID, ATM Card Number or Bank Account"
+msgstr ""
+"Loại proxy %s không được hỗ trợ cho khách hàng Việt Nam. Nó phải là "
+"ID người bán, số thẻ ATM hoặc tài khoản ngân hàng"
+
 #. module: l10n_vn
 #: model:account.report.line,name:l10n_vn.account_tax_report_line_02_01_vn
 msgid "Untaxed Purchase of Goods and Services"
 msgstr "Giá trị HHDV Mua vào"
 
 #. module: l10n_vn
 #: model:account.report.line,name:l10n_vn.account_tax_report_line_01_02_01_vn
-#: model:account.report.line,tag_name:l10n_vn.account_tax_report_line_01_02_01_vn
 msgid "Untaxed Purchase of Goods and Services taxed 0%"
 msgstr "Giá trị HHDV Mua vào chịu thuế 0%"
 
 #. module: l10n_vn
 #: model:account.report.line,name:l10n_vn.account_tax_report_line_03_02_01_vn
-#: model:account.report.line,tag_name:l10n_vn.account_tax_report_line_03_02_01_vn
 msgid "Untaxed Purchase of Goods and Services taxed 10%"
 msgstr "Giá trị HHDV Mua vào chịu thuế 10%"
 
 #. module: l10n_vn
 #: model:account.report.line,name:l10n_vn.account_tax_report_line_02_02_01_vn
-#: model:account.report.line,tag_name:l10n_vn.account_tax_report_line_02_02_01_vn
 msgid "Untaxed Purchase of Goods and Services taxed 5%"
 msgstr "Giá trị HHDV Mua vào chịu thuế 5%"
 
+#. module: l10n_vn
+#: model:account.report.line,name:l10n_vn.account_tax_report_line_04_02_01_vn
+msgid "Untaxed Purchase of Goods and Services taxed VAT Exemption"
+msgstr "Giá trị HHDV mua vào không chịu thuế GTGT"
```

Comment:
@aliyatastemirova Translation added but i notice the odoo version still in 14.0 in this PO/POT probably no effect at all and i'll keep it that way to avoid unecessary change right ?